<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CashierScreen</class>
 <widget class="QWidget" name="CashierScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>968</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cashier Screen</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="CheckOut">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="Warning">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <attribute name="title">
       <string>Date </string>
      </attribute>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="geometry">
        <rect>
         <x>-1</x>
         <y>2</y>
         <width>371</width>
         <height>291</height>
        </rect>
       </property>
      </widget>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>80</y>
         <width>421</width>
         <height>301</height>
        </rect>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;!! &lt;span style=&quot; font-weight:704; color:#ff0000;&quot;&gt;Warning&lt;/span&gt;: You are now in the danger zone. Any changes you make here may be irreversible.&lt;br /&gt;&lt;br /&gt;By using this account, you agree that you, the user, are authorized to access this area. Any change made using this area is&lt;br /&gt;your responsibility, and you shall account for any damage that may come as a result of accessing this area.&lt;br /&gt;&lt;br /&gt;If you are not authorized to view this area, please discontinue using this workstation and contact your supervisor&lt;br /&gt;&lt;span style=&quot; font-weight:704;&quot;&gt;IMMEDIATELY&lt;/span&gt;. Failure to comply will result in criminal prosecution to the fullest extent under the law.&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-weight:704;&quot;&gt;Go To next Tab to check out.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget">
      <attribute name="title">
       <string>Check_Out</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>941</width>
         <height>551</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="2">
         <widget class="QLineEdit" name="keyword">
          <property name="styleSheet">
           <string notr="true">QLineEdit{
border-radius: 5px;
}
</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QPushButton" name="addItemButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>Add Item</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="itemLabel">
          <property name="text">
           <string>Search By:</string>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="deleteItemButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>Delete Item</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QComboBox" name="filter">
          <property name="styleSheet">
           <string notr="true">QGroupBox{
border: 0;
}
QComboBox{
border-radius: 5px;
padding: 5px;
}
QComboBox::drop-down { 
	width: 0px; 
}</string>
          </property>
          <item>
           <property name="text">
            <string>Id</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Name</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="titleLabel">
          <property name="font">
           <font>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Cashier Screen</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="editItemButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>Edit Item</string>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QListWidget" name="cartListWidget"/>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="checkOutButton">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>Check Out</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="keywordLabel">
          <property name="text">
           <string>Id:</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QPushButton" name="ExitItem">
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>Exit Item</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="viewCartButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>Clear Cart</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPushButton" name="inventoryButton">
          <property name="styleSheet">
           <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
          </property>
          <property name="text">
           <string>Inventory</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="Information">
      <attribute name="title">
       <string>Check List</string>
      </attribute>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>431</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Did You took all money out of the draw?</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Did you took all reciept?</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>241</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>You Ready for log out?</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="LogOut">
      <attribute name="title">
       <string>Info</string>
      </attribute>
      <widget class="QPushButton" name="Exit">
       <property name="geometry">
        <rect>
         <x>680</x>
         <y>430</y>
         <width>241</width>
         <height>101</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>Exit</string>
       </property>
      </widget>
      <widget class="QPushButton" name="Total">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>201</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>Total Day</string>
       </property>
      </widget>
      <widget class="QPushButton" name="history">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>30</y>
         <width>181</width>
         <height>71</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>History Check Out</string>
       </property>
      </widget>
      <widget class="QPushButton" name="logout">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>120</y>
         <width>201</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>Log Out Account</string>
       </property>
      </widget>
      <widget class="QPushButton" name="changePassword">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>110</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>Change Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="profile">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>250</y>
         <width>181</width>
         <height>91</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton{
    color: white;
    background-color: #0e67ed;
    border-radius: 5px;
    padding: 5px;
}</string>
       </property>
       <property name="text">
        <string>Profile</string>
       </property>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>history</sender>
   <signal>clicked()</signal>
   <receiver>CashierScreen</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>501</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>483</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
